### PR TITLE
docs: add Synchronous Messages Fiddle example

### PR DESCRIPTION
**Pull request by 97amarnathk**
#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: no-notes

### CODE REVIEW

File: docs/fiddles/communication/two-processes/synchronous-messages/index.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+  </head>
+  <body>
+    <div>
+      <div>
+        <h1>Supports: Win, macOS, Linux <span>|</span> Process: Both</h1>
```
Comment (jkleinsc) [inline docs/fiddles/communication/two-processes/synchronous-messages/index.html]:
```suggestion
        <h1>Synchronous messages</h1>
        <i>Supports: Win, macOS, Linux <span>|</span> Process: Both</i>
```

File: docs/fiddles/communication/two-processes/synchronous-messages/main.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+const { app, BrowserWindow, ipcMain } = require('electron')
+
+let mainWindow = null
+
+function createWindow () {
+  const windowOptions = {
+    width: 600,
+    height: 400,
+    title: 'Synchronous',
```
Comment (jkleinsc) [inline docs/fiddles/communication/two-processes/synchronous-messages/main.js]:
```suggestion
    title: 'Synchronous messages',
```

Comment (97amarnathk) [reply docs/fiddles/communication/two-processes/synchronous-messages/main.js]: Okay. I will do this. Also for some reason the tests are failing on Circle CI Linux. What am I doing wrong?

Comment (jkleinsc) [reply docs/fiddles/communication/two-processes/synchronous-messages/main.js]: @97amarnathk don't worry about the failing tests on CircleCI Linux.  They are unrelated to your change.

Comment (97amarnathk) [reply docs/fiddles/communication/two-processes/synchronous-messages/main.js]: I have added the suggested changes.

File: docs/fiddles/communication/two-processes/synchronous-messages/index.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+  </head>
+  <body>
+    <div>
+      <div>
+        <h1>Synchronous messages</h1>
```
Comment (jkleinsc) [inline docs/fiddles/communication/two-processes/synchronous-messages/index.html]: I'd still include the "supports" message, but not as a H1. For example:
```suggestion
        <h1>Synchronous messages</h1>
        <i>Supports: Win, macOS, Linux <span>|</span> Process: Both</i>
```

**Comment by 97amarnathk** [comment]
> Looks better, but I would still include the "Supports" message
> I'd still include the "supports" message, but not as a H1. For example:
> 
> ```
>         <h1>Synchronous messages</h1>
>         <i>Supports: Win, macOS, Linux <span>|</span> Process: Both</i>
> ```

Isn't it implied since Electron is supposed to be cross platform?

File: docs/fiddles/communication/two-processes/synchronous-messages/index.html (hunk 3 above)
Comment (97amarnathk) [reply docs/fiddles/communication/two-processes/synchronous-messages/index.html]: @jkleinsc  I have pushed the changes requested.

**Pull request merged**